frontend/app/routes/api/health.ts

```diff
@@ -38,20 +30,31 @@ export async function loader({ context: { configProvider, serviceProvider, sessi
     includeComponents: toArray(include),
     includeDetails: isAuthorized(request),
     metadata: { buildId, version },
-    timeout: toNumber(timeout),
+    timeoutMs: toNumber(timeout),
   };
 
   //
   // TODO :: GjB ::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
   //
-  //   - add actual, non-dummy health checks
   //   - include details when isAuthorized()
   //   - cache execute() response for a short period (ie: rate-limiting)
   //
   // TODO :: GjB ::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
   //
 
-  const systemHealthSummary = await execute([dummyHealthCheck], healthCheckOptions);
+  const redisHealthCheck: HealthCheck = {
+    name: 'redis',
+    check: async () => {
+      await serviceProvider.getRedisService()?.ping();
+    },
+  };
+
+  // exclude the redis health check if the application is not using it
+  const useRedis = configProvider.getServerConfig().SESSION_STORAGE_TYPE === 'redis';
+  if (!useRedis) (healthCheckOptions.excludeComponents ??= []).push(redisHealthCheck.name);
```

Comment:
Use brackets and split it for readability.
```suggestion
  if (!useRedis) {
    healthCheckOptions.excludeComponents ??= []
    healthCheckOptions.excludeComponents.push(redisHealthCheck.name);
  }
```

Reply:
[Fixed](https://github.com/DTS-STN/canadian-dental-care-plan/pull/2427/commits/c7355e9ed4037b3f22f17511f5611daebbfcfab9#diff-474c19e6b1da99eeef162fb49a9bb8d7b1bb66ed332785620259c1781135b80bR54-R57)